pex/common.py

```diff
@@ -482,9 +484,13 @@ def _normalize(self, dst):
         return dst
 
     def _check_tag(self, fn, label):
-        for fs_label, fs in self.filesets.items():
-            if fn in fs and fs_label != label:
-                raise self.ChrootTaggingException(fn, fs_label, label)
+        """Raises ChrootTaggingException if a file was added under more than one label."""
+        fs_label = self._file_index.get(fn, None)
```

Comment:
It looks like this works, but it's clunkier than need be. Maybe:
```
$ python2.7
Python 2.7.18 (default, Jul  1 2022, 10:30:50)
[GCC 11.2.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> x = {}
>>> def check(key, label):
...   if label != x.setdefault(key, label):
...     raise Exception("{} already labeled as {}".format(key, label))
...
>>> check("foo", "bar")
>>> check("foo", "baz")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 3, in check
Exception: foo already labeled as baz
>>>
```

Reply:
Um, my quick repl session obviously has a bug in the exception message, but you get the point.

Reply:
Done!